### PR TITLE
Replace httptest with http, fix regtest.json data file

### PR DESCRIPTION
**Pull request by udit-gulati**
## Description

1. Uses `http` in place of `httptest` when starting server.
2. Fix the `data/regtest.json` data file, replace block hashes with block headers hex.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
    
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -44,12 +42,9 @@ func main() {
 	btcLC := btclightclient.NewBTCLightClientWithData(&chaincfg.MainNetParams, headers, int(startHeight))
 	btcLC.Status()
 
-	rpcService := rpcserver.NewRPCServer(btcLC)
-	log.Info().Msgf("RPC server running at: %s", rpcService.URL)
-
-	// Create channel to listen for interrupt signal
-	// Wait for interrupt signal
-	sigChan := make(chan os.Signal, 1)
-	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM)
```
[the remark this answers is not in the document]
Comment (udit-gulati) [reply main.go:53]: `http.ListenAndServe()` is blocking while `http.NewServer()` was not

File: main.go [hunk 2]
```diff
@@ -44,12 +42,9 @@ func main() {
 	btcLC := btclightclient.NewBTCLightClientWithData(&chaincfg.MainNetParams, headers, int(startHeight))
```
Comment (vuvoth) [inline main.go:42]: Let add one parameter to run btclightclient for networkconfig

File: data/regtest.json [hunk 3]
```diff
@@ -1,15 +1,25 @@
 {
-    "start_height": 11,
+    "start_height": 1,
     "header_strings": [
```
Comment (vuvoth) [inline data/regtest.json:3]:
```suggestion
    "blockheaders": [
```

**Review by vuvoth — APPROVED** [review]
LGTM!

**Pull request merged**